triggers.gs

```diff
@@ -0,0 +1,63 @@
+/**
+ * Trigger for the rule for the number of executions for the script
+ */
+function mostExecutedTrigger() {
+  var files = DriveApp.getFilesByName(adminRuleFileName);
```

Comment:
Can you handle the case if adminRuleFileName doesn't exists.

Reply:
If it doesn't exist then function won't do anything

Reply:
Can you please confirm that no exception is being thrown.

Reply:
Yeah
I tried once deleting the admin rule file too